public/javascripts/tests.js

```diff
@@ -84,17 +100,105 @@ function renderTestsList(jobs, is_operator, restart_url) {
 	    },
 	],
     } );
+
+    var scheduled_table = $('#scheduled').DataTable( {
+        "pagingType" : 'simple',
+        "order": [[3, 'asc'], [0, 'asc']],
+        "ajax": {
+            "url": "/tests/list_ajax",
+            "type": "POST", // we use POST as the URLs can get long
+            "data": function(d) {
+            var ret = {
+                "relevant": $('#relevantfilter').prop('checked'),
+                "state" : 'scheduled'
+            };
+            if (jobs != null) {
+                ret['jobs'] = jobs;
+                ret['initial'] = 1;
+            }
+            // reset for reload
+            jobs = null;
+            return ret;
+            }
+        },
+        "columns": [
+            { "data": "name" },
+            { "data": "test" },
+            { "data": "deps" },
+            { "data": "priority" },
+        ],
+        "columnDefs": [
+            { targets: 0,
+                className: "name",
+                "render": function ( data, type, row ) {
+                var name = 'Build' + row['build'];
+                name += " of ";
+                return name + row['distri'] + "-" + row['flavor'] + "." + row['arch'];
+                },
+            },
+            { targets: 1,
+                className: "test",
+                "render": function ( data, type, row ) {
+                    if (type === 'display') {
+                        var url = test_url.replace('REPLACEIT', row['id']);
+                        return '<a class="overview_' + row['result'] + '" href="' + url + '">' + data + '</a>';
+                    } else {
+                        return data;
+                    }
```

Comment:
these functions should be extracted I'd say - they look pretty much the same for the 2 tables.